src/patternfly/components/InputGroup/examples/input-group-example.hbs

```diff
@@ -96,7 +96,7 @@
 {{#> input-group}}
   {{#> form-control controlType="input" input="true" form-control--attribute='type="text" id="textInput10" name="textInput10" aria-label="input example with popover"'}}
```

Comment:
super nit - would you mind sentence casing these `aria-label` values?

```suggestion
  {{#> form-control controlType="input" input="true" form-control--attribute='type="text" id="textInput10" name="textInput10" aria-label="Input example with popover"'}}
```

Reply:
Updated the file, most of the aria-labels weren't capitalized